VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitNavigationActivity.java

```diff
@@ -163,8 +163,7 @@ private void setCurrentUnit(CourseComponent component){
         resultData.putExtra(Router.EXTRA_COURSE_COMPONENT_ID, courseComponentId);
         setResult(RESULT_OK, resultData);
 
-        boolean isPortrait = getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT;
-        environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId(), isPortrait);
+        environment.getSegment().trackCourseComponentViewed(selectedUnit.getId(), courseData.getCourse().getId());
```

Comment:
I am generally against creating unnecessary functions and this call will only be made 2 times now that we have removed one in that `if-else` case scenario.